Print_A_Bot/djwifi/templates/system/access_point/default_hostapd.conf

```diff
@@ -0,0 +1,20 @@
+# Defaults for hostapd initscript
```

Comment:
everything under 

` Print_A_Bot/djwifi/templates/system` is a template to manage a linux service